packages/account/src/Sections/Security/DeactivateAccount/account-has-balance.jsx

```diff
@@ -122,6 +134,31 @@ const AccountHasBalanceOrOpenPositions = ({ details, mt5_login_list, client_acco
                         ))}
                     </Wrapper>
                 )}
+                {!!account_pending_withdrawals.length && (
+                    <Wrapper title={localize('Pending withdrawal request:')}>
+                        <Text as='p'>
+                            {localize(
+                                'We are still processing your withdrawal request.\nPlease wait for the transaction to be completed before deactivating your account.'
+                            )}
+                        </Text>
+                        {account_pending_withdrawals.map(account => (
+                            <Content
+                                key={account.loginid}
+                                currency_icon={`IcCurrency-${account.icon}`}
+                                loginid={account.loginid}
+                                title={account.title}
+                                value={
+                                    <Localize
+                                        i18n_default_text='{{pending_withdrawals}}'
+                                        values={{ pending_withdrawals: account.withdrawals }}
+                                    />
+                                }
+                            />
+                        ))}
+                    </Wrapper>
+                )}
+                {/* {!!deriv_balance.length && ( */}
```

Comment:
Do we need to keep this commented code?

Reply:
Sorry this was supposed to be a draft. Will remove it.